.tekton/pull-request.yaml

```diff
@@ -190,11 +216,12 @@ spec:
             description: "custom bundle for fbc-builder pipeline"
           steps:
             - name: build-bundles
-              image: quay.io/konflux-ci/pull-request-builds:appstudio-utils-{{revision}}
+              image: $(params.image-target)
               workingDir: $(workspaces.source.path)/source
               script: |
                 #!/usr/bin/env bash
                 set -e
+                git config --global --add safe.directory /workspace/source/source
```

Comment:
I needed to add this because we were hitting an issue when trying to run the included script. This started happening after I changed the buildah task from the custom one for appstudio-utils to our buildah-remote task. I suspect that the issue is that the buildah-remote task has the user defined: https://github.com/konflux-ci/build-definitions/blob/main/task/buildah-remote/0.4/buildah-remote.yaml#L1120-L1122